xapian-core/configure.ac

```diff
@@ -1216,6 +1217,31 @@ if test yes = "$enable_log"; then
     [Define if you want a log of methods called and other debug messages])
 fi
 
+dnl Check if IEEE-754 is followed for representing floating point numbers by the platform.
+AC_RUN_IFELSE([
+  AC_LANG_PROGRAM(
+    [[
+#include <cstdint>
+#include <cstring>
+using namespace std;
+    ]],
+    [[
+      double a = 2353.3523423;
+      uint64_t bit_pattern =
+      0b0100000010100010011000101011010001100110001101011011111011111111;
+      if (memcmp(&bit_pattern, &a, sizeof(double)) == 0) return 0;
+      return 1;
```

Comment:
Won't this test always fail on a big endian platform?

Reply:
Ah no, it works so long as the integers and `double` have the same endianness.

In theory the two could differ I think, but I don't know of any platform where they actually do so slow handling for that case isn't a big deal.